src/containers/Tenant/Diagnostics/Configs/Configs.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+
+import type {Column} from '@gravity-ui/react-data-table';
+import {Switch} from '@gravity-ui/uikit';
+import {StringParam, useQueryParam} from 'use-query-params';
+
+import {ResizeableDataTable} from '../../../../components/ResizeableDataTable/ResizeableDataTable';
+import {Search} from '../../../../components/Search';
+import {TableWithControlsLayout} from '../../../../components/TableWithControlsLayout/TableWithControlsLayout';
+import {setFeatureFlagsFilter, tenantApi} from '../../../../store/reducers/tenant/tenant';
+import type {TClusterConfigFeatureFlag} from '../../../../types/api/cluster';
+import {useTypedDispatch, useTypedSelector} from '../../../../utils/hooks';
+
+const FEATURE_FLAGS_COLUMNS_WIDTH_LS_KEY = 'featureFlagsColumnsWidth';
+
+const columns: Column<TClusterConfigFeatureFlag>[] = [
+    {
+        name: 'Feature flag',
+        render: ({row}) =>
+            row.Current ? <b title="Default value was changed">{row.Name}</b> : row.Name,
+        width: 400,
+        sortable: false,
+    },
+    {
+        name: 'Default',
+        render: ({row}) => {
+            switch (row.Default) {
+                case true:
+                    return 'Enabled';
+                case false:
+                    return 'Disabled';
+                default:
+                    return '-';
+            }
+        },
+        width: 100,
+        sortable: false,
+        resizeable: false,
+    },
+    {
+        name: 'Current',
+        render: ({row}) => <Switch disabled checked={(row.Current ?? row.Default) || false} />,
+        width: 100,
+        sortable: false,
+        resizeable: false,
+    },
+];
+
+interface Props {
+    path: string;
+    database: string;
+}
+
+export const Configs = ({database}: Props) => {
+    const dispatch = useTypedDispatch();
+    const [search, setSearch] = useQueryParam('search', StringParam);
+    const {currentData = [], isFetching, isError} = tenantApi.useGetClusterConfigQuery({database});
+    const featureFlagsFilter = useTypedSelector(
+        (state) => state.tenant.featureFlagsFilter,
+    )?.toLocaleLowerCase();
+
+    // sync store and query filter
+    React.useEffect(() => {
+        if (search) {
+            dispatch(setFeatureFlagsFilter(search));
+        } else if (featureFlagsFilter) {
+            setSearch(featureFlagsFilter);
```

Comment:
Please, add `replaceIn` as second param, currently it pushes new entries in history, so if you click back button, you will go to the previous search value, not the previous page or tab